.github/workflows/conformance.yml

```diff
@@ -176,6 +177,8 @@ jobs:
       run:
         shell: bash
     needs: generate-matrix
+    env:
+      UUID: ${{github.run_id}}
```

Comment:
github.run_id is not a UUID however, is it? Can you use a different name so it's not misleading?

Reply:
Sure, I can rename it to simply RUN_ID.